main.hoc

```diff
@@ -1,5 +1,6 @@
 // Hyperparameters 
-N_RUNS = 10 // Number of runs per group
+N_RUNS = 1 // Number of runs per group
+PP_freq_ = 3 //
```

Comment:
important to define this in comments

Reply:
WAIT is PP_freq_ a global var? If so, I would avoid that, and rather make it an argument into `DentateGyrus`